### DISCUSSION TITLE
Support for Jobs plugin and itemsadder custom blocks

### DISCUSSION THREAD

**Discussion by AxDevv** [BG-Software-LLC/WildTools#205] [Pending Feature Requests]
it would be really useful if the items worked with **Jobs**, because currently we don't gain any exp or currency with them, 

And we should also make the custom **ItemsAdder** blocks compatible with the trenchPickaxe

**Comment by OmerBenGera** [comment]
You can add the Jobs plugin to the `other-plugins` list in the config file.
Regarding ItemsAdder, I am not sure what exactly breaks, but try adding it to the `claiming-plugins` or the `other-plugins` list in the config file as well.
